examples/log-source-app/log-source-common/log-source-app.matter

```diff
@@ -442,15 +783,48 @@ endpoint 0 {
     ram      attribute clusterRevision default = 1;
   }
 
+  server cluster BasicInformation {
+    callback attribute dataModelRevision default = 10;
+    callback attribute vendorName;
```

Comment:
Are all these attributes that are being enabled actually implemented in these apps?